com.wetgenes.gamecake.yaml

```
@@ -0,0 +1,35 @@
+app-id: com.wetgenes.gamecake
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gamecake
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+modules:
+  - name: luajit
+    no-autogen: true
+    make-install-args:
+      - PREFIX=/app
+    post-install:
+      - ln -sf ./luajit-2.1.0-beta3 /app/bin/luajit
+    sources:
+      - type: archive
+        url: https://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz
+        sha256: 1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3
+  - name: gamecake
+    buildsystem: simple
+    build-commands:
+      - ./make
+      - cp ./exe/gamecake.nix /app/bin/gamecake
+      - install -D com.wetgenes.gamecake.png --target-directory=/app/share/icons/hicolor/512x512/apps/
```

the icon is 256x256.
```suggestion
      - install -D com.wetgenes.gamecake.png --target-directory=/app/share/icons/hicolor/256x256/apps/
```

---

the file is never referenced so it is not found. Same for the next 2 lines.